vision_datasets/common/data_manifest.py

```diff
@@ -866,6 +881,8 @@ def process_labels_without_categories(image):
                 label = [c_id] + bbox
                 img.labels_extra_info['iscrowd'] = img.labels_extra_info.get('iscrowd', [])
                 img.labels_extra_info['iscrowd'].append(annotation.get('iscrowd', 0))
+            elif 'query' in annotation:
```

Comment:
interesting, the previous 'text' etc. keys are not processed here?

Reply:
no, for IMCAP and and IMAGE_TEXT_MATCHING the function returns earlier